structure/0006-governance.md

```diff
@@ -2,53 +2,135 @@
+In general, a team is responsible for performing the tasks outlined in the RFC that proposed its creation.
+
+Most teams will also maintain one or more repositories on GitHub.
+The team is responsible for reviewing pull requests and ensuring that they meet the Quilt projects goals in quality and purpose.
+Once a team has decided that a pull request is suitable for merging, a team lead will be responsible for the actual merging.
+
+### Creating a Team
+
+A new team can be created by passing an RFC proposing that team.
+The initial team leads must be specified in the RFC and must approve of the RFC before it is merged.
+Once the [RFC process](0001-rfc-process.md) has concluded and the RFC was merged,
+a new team with the specified leads will be created on GitHub by the Administrative Board.
+
+The RFC proposing the new team should contain the following major sections:
+
+- **Motivation**:
+  - Why is this team needed?
+  - What problems will this team solve?
+  - What tasks will this team be responsible for?
+
+- **Explanation**:
+  - What will this team *do*?
+  - Who will be the team leads?
+  - What new projects will be created that this team will be responsible for?
+  - How will each of these projects benefit the QuiltMC organization/community?
+  - What processes will this team follow for each of the different kinds of changes their projects may undergo?
+
+- **Drawbacks**:
+  - Is there overlap of responsibilities?
+  - Is there excessive granularity?
+
+- **Rationale and Alternatives**:
+  - Is there already a team responsible for the described tasks?
+  - If this RFC doesn't pass, who would be responsible for the described tasks?
+
+- **Prior Art**:
+  - Are there other organizations or projects with comparable tasks?
+
+### Sub Teams
 
-The vote must be initiated with a detailed call to action, describing why the staff member feels that the person should be removed from power. The accused will have one week to submit a detailed counterargument, at which point the vote will be called.
+Some teams may wish for more granular organization of their structure.
+In those cases, these teams may create sub teams.
 
-### The Administrative Board
-The administrative board is the group of staff that holds ultimate power over the direction of the project. It is made up of three or more members so as to prevent deadlocks in the decision-making process. Members of the administrative board may be members of other teams within the project as well. Admins do not generally make use of their position unless a critical decision needs to be made whose team is unable to come to a consensus. In this case, the board will debate internally until they come to a decision. The administrative board acts primarily as a tiebreaker.
+Sub teams behave like regular teams, except they are not required to have a team lead.
+If the team doesn't have its own team lead, the parent's team leads act as its team lead.
```

Comment:
"If the team doesn't have its..." -> "If the sub team doesn't have its..."

"...parent's team leads act..." -> "...parent team's lead acts..."